### PR TITLE
chore: Address sonar warnings

### PR DESCRIPTION
**Pull request by mattcole19**
Some work for [DI-2348](https://issues.corp.twilio.com/browse/DI-2348) - Addressing all easily actionable warnings from sonarlint for all generators.

### CODE REVIEW

File: examples/go/go-client/terraform/resources/api/v2010/README.md [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
-**recording_status_callback_event** | list(string) | Optional | 
-**sid** | string | *Computed* | 
-**status** | string | Optional | 
-**pause_behavior** | string | Optional | 
+**x_twilio_webhook_enabled** |  | Optional | 
+**recording_status_callback** |  | Optional | 
+**recording_status_callback_event** |  | Optional | 
+**sid** |  | *Computed* | 
+**status** |  | Optional | 
+**pause_behavior** |  | Optional |
```
Comment (childish-sambino) [inline examples/go/go-client/terraform/resources/api/v2010/README.md:13]: This doesn't look right.

Comment (mattcole19) [reply examples/go/go-client/terraform/resources/api/v2010/README.md:13]: Hmm.. sonarlint said the getFullType method doesn't get used, but I guess it gets used under the hood. I'll add it back in.

Just for my understanding - is this due to the lombok.value annotation?

Comment (childish-sambino) [reply examples/go/go-client/terraform/resources/api/v2010/README.md:13]: Sonar doesn't know it's being used in the mustache template.

Comment (mattcole19) [reply examples/go/go-client/terraform/resources/api/v2010/README.md:13]: Ah didn't know it got used there, that makes sense.

**Review by childish-sambino — APPROVED** [review]

**Pull request merged**